packages/opentelemetry-core/src/common/attributes.ts

```diff
@@ -13,30 +13,40 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { SpanAttributeValue, SpanAttributes } from '@opentelemetry/api';
 
-export function sanitizeAttributes(attributes: unknown): SpanAttributes {
-  const out: SpanAttributes = {};
+import { diag, AttributeValue, Attributes } from '@opentelemetry/api';
 
-  if (attributes == null || typeof attributes !== 'object') {
+export function sanitizeAttributes(attributes: unknown): Attributes {
+  const out: Attributes = {};
+
+  if (typeof attributes !== 'object' || attributes == null) {
     return out;
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-  for (const [k, v] of Object.entries(attributes!)) {
-    if (isAttributeValue(v)) {
-      if (Array.isArray(v)) {
-        out[k] = v.slice();
-      } else {
-        out[k] = v;
-      }
+  for (const [k, v] of Object.entries(attributes)) {
```

Comment:
nit pick but i would prefer using key / value instead of single letter variable for readability, WDYT ?

Reply:
SGTM. updated :)